tests/functional/modules/test_zos_lineinfile_func.py

```diff
@@ -41,7 +44,7 @@
 call_c_jcl="""//PDSELOCK JOB MSGCLASS=A,MSGLEVEL=(1,1),NOTIFY=&SYSUID,REGION=0M
 //LOCKMEM  EXEC PGM=BPXBATCH
 //STDPARM DD *
-SH /tmp/disp_shr/pdse-lock '{0}({1})'
+SH {2}pdse-lock '{0}({1})'
```

Comment:
you don't have to change this now, but for the purposes of future maintainability, it would be better to have the {x}s  in order. Imagine coming back to this code block in the future and trying to figure out why the order {2} {0} {1} was selected and whether that order was significant or random.

Reply:
agree